core/src/script/CGXP/widgets/tree/LayerTree.js

```diff
@@ -404,6 +411,17 @@ cgxp.tree.LayerTree = Ext.extend(Ext.tree.TreePanel, {
 
                 var timeWidget;
                 if (item.time) {
+                    if (this.initialState && this.initialState['time_' + item.name]) {
+                      var times = this.initialState['time_' + item.name].split('/');
+                      if (times.length === 2) {
```

Comment:
Yes but I think that's `tree_time_myGroupA=2016-01-01` should also be valid, to be verified :-)